build.gradle

```diff
@@ -17,6 +17,7 @@ dependencies {
     implementation 'org.json:json:20231013'
     implementation 'org.apache.httpcomponents:httpclient:4.5.14'
     implementation 'com.nimbusds:oauth2-oidc-sdk:10.14.2'
+    implementation 'org.projectlombok:lombok:1.18.28'
```

Comment:
You need all 4 of these dependencies:

https://github.com/factset/enterprise-sdk-eventdriven-factsettrading-java/blob/main/build.gradle.kts#L34-L37